### DISCUSSION TITLE
Any news on the pretrained entity linking model?

### DISCUSSION THREAD

**Discussion by HenryDashwood** [explosion/spaCy#6665] [Language Support]
Hi there!

Apologies if this is being discussed elsewhere. I was just wondering if there had been any developments in making the pretrained entity linking models available? See here for the last discussion I can find about them: #5156 

Many thanks!

**Comment by svlandeg** [comment]
This is not currently in the pipeline. The main issue is that it's simply hard to decide what a pretrained EL model should do. Which data should it be trained on? Which knowledge base should it use? Which type of entities should it try to resolve?

Prior probabilities can vary so much between different domains/text sources/communities, that it's a huge challenge to define a model that would be generalizable. It's a little like Text categorization. We currently don't provide pretrained models, but we provide all the tools for you to be able to train your own on your specific use-case.

**Comment by HenryDashwood** [comment]
No problem. That's good to know!
